src/cli/coinhourbank.go

```diff
@@ -0,0 +1,148 @@
+}
+
+func transferCoinhoursCmd() *cobra.Command {
+	transferCoinhoursCmd := &cobra.Command{
+		Use:          "transferCoinhours [destination address] [hours amount]",
+		Short:        "Transfer coinhours from one coinhour bank account to another",
+		Long:         `Transferring coinhours from one account to another does not require any transaction fee.`,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(2),
+		RunE: func(c *cobra.Command, args []string) error {
+			bankClient, err := getCoinhourBankClient(c)
+			if err != nil {
+				return err
+			}
+
+			address, _ := c.Flags().GetString("address")
+
+			coinhours, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			return bankClient.TransferHours(chb.Account(address), chb.Account(args[1]), chb.CoinHours(coinhours))
+		},
+	}
+
+	transferCoinhoursCmd.Flags().StringP("wallet-file", "f", "", "[wallet file or path] From wallet. If no path is specified your default wallet path will be used.")
+	transferCoinhoursCmd.Flags().StringP("password", "p", "", "wallet password")
+	transferCoinhoursCmd.Flags().StringP("address", "a", "", "coinhour bank to take coinhours from")
+	transferCoinhoursCmd.Flags().StringP("nodeURL", "n", "http://localhost:6420", "skycoin node url")
+	transferCoinhoursCmd.Flags().StringP("bankURL", "b", "http://localhost:8081", "coinhour bank backend url")
+
+	return transferCoinhoursCmd
+}
+
+func withdrawCoinhoursCmd() *cobra.Command {
+	withdrawCoinhoursCmd := &cobra.Command{
+		Use:          "withdrawCoinhours [hours amount]",
+		Short:        "Withdraws coinhours from coinhour bank.",
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE: func(c *cobra.Command, args []string) error {
+			bankClient, err := getCoinhourBankClient(c)
+			if err != nil {
+				return err
+			}
+
+			address, _ := c.Flags().GetString("address")
+
+			coinhours, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			return bankClient.WithdrawHours(chb.CoinHours(coinhours), chb.Account(address))
+		},
+	}
+
+	withdrawCoinhoursCmd.Flags().StringP("wallet-file", "f", "", "[wallet file or path] From wallet. If no path is specified your default wallet path will be used.")
+	withdrawCoinhoursCmd.Flags().StringP("password", "p", "", "wallet password")
+	withdrawCoinhoursCmd.Flags().StringP("address", "a", "", "wallet address to take coinhours from")
+	withdrawCoinhoursCmd.Flags().StringP("nodeURL", "n", "http://localhost:6420", "skycoin node url")
+	withdrawCoinhoursCmd.Flags().StringP("bankURL", "b", "http://localhost:8081", "coinhour bank backend url")
+
+	return withdrawCoinhoursCmd
+}
+
+func getCoinhourBankClient(c *cobra.Command) (*client.HourBankClient, error) {
+	walletFile, err := c.Flags().GetString("wallet-file")
+	if err != nil {
+		return nil, err
+	}
+
+	wlt, err := resolveWalletPath(cliConfig, walletFile)
+	if err != nil {
+		return nil, err
+	}
+
+	wltPassword, err := c.Flags().GetString("password")
```

Comment:
Wallet passwords need to use the special password reader so that it does not need to be entered in the terminal